### ISSUE TITLE
Snotgobbler dropped a wedding ring.

### ISSUE THREAD

**Issue by AudBobb** [CleverRaven/Cataclysm-DDA#77280]
### Describe the bug

Snotgobbler, a child zombie, dropped a wedding ring. Went through all item groups in the monster's drop files and could not find any groups including a wedding ring, so not sure how this happened.

### Attach save file

[Sayner.zip](https://github.com/user-attachments/files/17495616/Sayner.zip)

### Steps to reproduce

Attempted to reproduce by loading test world and debug spawn and kill snotgobblers. Spawned 482 snotgobblers, and was not able to reproduce. I'm sure there is a faster way to simulate their drops using a script, but I don't know how. 

### Expected behavior

Children aren't married.

### Screenshots

Log from when this occured. I was travelling through unexplored forest, and this is the only enemy that was in the area, so no chance it the ring dropped from another creature.
![image](https://github.com/user-attachments/assets/82ebda9d-c0d6-41ce-8324-c1f37c686977)
![image](https://github.com/user-attachments/assets/eefe0840-0c10-4be7-aab5-e0d7cfe6e84b)

Picture of attempt to reproduce in controlled environment, with resulting ring drops.
![image](https://github.com/user-attachments/assets/8b94466e-edf2-43c8-9a47-fedd52ecdf48)

### Versions and configuration

Windows 10
Game version - 2024-10-23-0605 5d32050, with custom UI change that doesn't touch any code related to monsters, itemgroups, or anything that should affect this.
Mods - Magiclysm

**Comment by Night-Pryanik** [comment]
Wedding rings were added to most ordinary zombie drops in #76852. Though its author assured us that zombie children shouldn't drop wedding rings as their drops override ordinary zombie drops.

**Comment by AudBobb** [comment]
I am aware, that's why I wanted to bring this up. I looked through the json files following all the nested itemgroups but couldn't find any that contained a wedding ring, so not sure why the wedding ring was dropped, but there really was no room for uncertainty in that the snot gobbler definitely dropped the wedding ring. I had just unloaded everything from my inventory before heading out, and this was the first enemy I encountered, and I encountered it on it's own with no other enemies around.
